.github/workflows/alias-release.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: 🛠️ Update release alias tags
+
+run-name: Update alias for ${{ github.event.action }} ${{ github.event.release.name }}
+
+on:
+  release:
+    types:
+      - published
+      - deleted
+
+permissions:
+  actions: read
+  contents: write
+
+jobs:
+  update-alias:
+    uses: uclahs-cds/tool-create-release/.github/workflows/wf-alias-release.yaml@v1
+    # Secrets are only required until tool-create-release is made public
+    secrets: inherit
```

Comment:
**question (non-blocking)**: Can these be removed now that tool-create-release is public?

Reply:
We'll have to see - since I originally added that comment I started using our secret token to create the releases, so I'll have to play with it. I'll figure it out in the next few days and either remove the use of secrets or the comment saying that it's temporary.